INSTALL.adoc

```diff
@@ -20,6 +20,9 @@
 * Under Cygwin, the `gcc-core` package is required. `flexdll` is also necessary
   for shared library support.
 
+* Binutils including `ar`, `ranlib`, and `strip` are required if your
+  distribution doesn't already provide them with the C compiler.
```

Comment:
The written form "does not" would be more consistent with the rest of the document. (A "won't" already snuck in, but otherwise the long form is used consistently.)